Update microsoft-defender-endpoint-ios.md (#6128)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-ios.md

- Access to [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app to enrolled user groups in your organization.

    > [!NOTE]

>     > Microsoft Defender for Endpoint now extends support to organization’s data within a managed application for those who aren’t using an MDM but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [Mobile Application Management (MAM).](https://docs.microsoft.com/en-us/mem/intune/apps/mam-faq)

Reviewer: ```suggestion
    > Microsoft Defender for Endpoint now extends support to an organization’s data within a managed application for those who aren’t using mobile device management (MDM) but are using Intune to manage mobile applications. It also extends this support to customers who use other enterprise mobility management solutions, while still using Intune for [mobile application management (MAM)](/mem/intune/apps/mam-faq).
```

> >Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune mobile device management (MDM) only.  

Reviewer: ```suggestion
    > Prior to this update, Microsoft Defender for Endpoint worked on devices that were enrolled using Intune MDM only.  
```

- [Configure Microsoft Defender for Endpoint on iOS features](ios-configure-features.md)
- [Configure app protection policy to include Defender for Endpoint risk signals (MAM)](ios-install-unmanaged.md)
- [Configure Conditional Access policy based on device risk score from Microsoft Defender for Endpoint](ios-configure-features.md#conditional-access-with-defender-for-endpoint-on-ios)

> - [What is mobile app management in Microsoft Intune? | How to get started with MAM.](https://docs.microsoft.com/en-us/mem/intune/apps/app-management#mobile-application-management-mam-basics)

Reviewer: ```suggestion
- [Mobile Application Management (MAM) basics](/mem/intune/apps/app-management#mobile-application-management-mam-basics)
```
